locksmith/src/controllers/userController.ts

```diff
@@ -27,14 +27,31 @@ namespace UserController {
     res: Response
   ): Promise<any> {
     let emailAddress = req.params.emailAddress
-    let result = await UserOperations.getUserPrivateKeyByEmailAddress(emailAddress)
+    let result = await UserOperations.getUserPrivateKeyByEmailAddress(
+      emailAddress
+    )
 
     if (result) {
       return res.json({ passwordEncryptedPrivateKey: result })
     } else {
       return res.sendStatus(400)
     }
   }
+
+  export async function retrieveRecoveryPhrase(
+    req: Request,
+    res: Response
+  ): Promise<any> {
+    let emailAddress = req.params.emailAddress
+    let result = await UserOperations.getUserRecoveryPhraseByEmailAddress(emailAddress)
+
+
+    if (result) {
+      return res.json({ recoveryPhrase: result })
+    } else {
+      return res.sendStatus(400)
+    }
+  }
 }
 
-export = UserController
\ No newline at end of file
+export = UserController
```

Comment:
I think we have linting/prettifier errors in the locksmith project that we need to solve...

Reply:
will chat with Chris about this on monday!